backend/src/main.py

```diff
@@ -0,0 +1,56 @@
+
+# General imports =================================================================================
+import os, sys, git
+import multiprocessing as mp
+
+# Project specific imports ========================================================================
+git_repo = git.Repo(__file__, search_parent_directories=True).git.rev_parse("--show-toplevel")
+sys.path.insert(0, os.path.join(git_repo, "backend", 'proto/Python'))
+sys.path.insert(0, os.path.join(git_repo, "backend"))
```

Comment:
Apparently git is not good for long running processes as it leaks resources. https://gitpython.readthedocs.io/en/stable/intro.html#limitations

What do we need this for anyway?

Reply:
swapped for basic os operations